docs/prob/api-client/unexpected-api-client-read-error.md

```diff
@@ -0,0 +1,26 @@
+# Problem: Unexpected API client read error
```

Comment:
Remember to add all these to the `_sidebar.md` file as well :)

Reply:
Right!